user/links/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "links"
+pkgver = "2.30"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr", "--mandir=PREFIX/share/man"]
+hostmakedepends = [
+    "libjpeg-turbo-devel",
+    "libtiff-devel",
+    "openssl3-devel",
+    "zlib-ng-compat-devel",
+]
```

Comment:
libraries to be built against go in `makedepends`
```suggestion
makedepends = [
    "libjpeg-turbo-devel",
    "libtiff-devel",
    "openssl3-devel",
    "zlib-ng-compat-devel",
]
```

Reply:
also forgor to mention but `pkgconf` should be in `hostmakedepends` as well